jaxrs-jwt/generateKeys.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+echo "Generating private key..."
+ssh-keygen -t rsa -b 2048 -f temp.pem
```

Comment:
This is not acceptable. This is a script that will only work on linux (under some conditions)
we need a java-only solution. java's keytool could probably help here.

Reply:
@ctomc I don't thing that keytool is capable of doing all the work, openssl would still be required to make both keys in correct format. What if I wrote 2 scripts - 1 for windows and 1 for unix, that would use keytool + openssl? Would that be acceptable?